### PR TITLE
Create initial blog app

### PR DESCRIPTION
**Pull request by igorkruglyak**

### CODE REVIEW

File: blog/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+"""Migration file."""
```
Comment (webknjaz) [inline blog/migrations/0001_initial.py]: Just exclude autogenerated files from pre-commit checks. There's no point in reformatting them.

File: blog/migrations/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+"""Django website."""
```
Comment (webknjaz) [inline blog/migrations/__init__.py]: No, it's DB migrations, not a website.

File: blog/models.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    created_date = models.DateTimeField(default=timezone.now)
+    published_date = models.DateTimeField(blank=True, null=True)
+
+    def publish(self):
+        """Publish date method."""
+        self.published_date = timezone.now()
+        self.save()
+
+    def __str__(self):
+        """Return title in admin."""
```
Comment (webknjaz) [inline blog/models.py]: This method is called when the object is being rendered into a string. Read https://docs.python.org/3/reference/datamodel.html for more details.
Don't state obvious things like "returns something" when it's literally a single instruction in this method and it's just one line below.

File: blog/models.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
+"""Models for Blog app."""
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+
+class Post(models.Model):
+    """Create Post model for Blog app."""
```
Comment (webknjaz) [inline blog/models.py]: It's a class, not a function. So there MUSTN'T be any action here.

File: blog/models.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE,
+    )
+    title = models.CharField(max_length=200)
+    text = models.TextField()
+    created_date = models.DateTimeField(default=timezone.now)
+    published_date = models.DateTimeField(blank=True, null=True)
+
+    def publish(self):
+        """Publish date method."""
```
Comment (webknjaz) [inline blog/models.py]: What does it do?

File: blog/models.py (hunk 4 above)
Comment (igorkruglyak) [reply blog/models.py]: I should write """Post model for Blog app."""?

File: blog/migrations/__init__.py (hunk 2 above)
Comment (igorkruglyak) [reply blog/migrations/__init__.py]: I just have to delete? No need to write here?

File: blog/models.py (hunk 5 above)
Comment (igorkruglyak) [reply blog/models.py]: Saves the publication date?

File: blog/models.py (hunk 3 above)
Comment (igorkruglyak) [reply blog/models.py]: How should I write correctly?

File: blog/models.py (hunk 4 above)
Comment (webknjaz) [reply blog/models.py]: that'd work

File: blog/models.py (hunk 5 above)
Comment (webknjaz) [reply blog/models.py]: No, it publishes the post.

File: blog/models.py (hunk 3 above)
Comment (webknjaz) [reply blog/models.py]: Render post as a string.

File: blog/migrations/__init__.py (hunk 2 above)
Comment (webknjaz) [reply blog/migrations/__init__.py]: Just write that it's migrations. Because the current docstring is a lie.

File: blog/models.py (hunk 5 above)
Comment (igorkruglyak) [reply blog/models.py]: I should write """Publish the post.""" ?

Comment (webknjaz) [reply blog/models.py]: yes

Comment (webknjaz) [reply blog/models.py]: or "mark post as published"

**Review by webknjaz — APPROVED** [review]

**Pull request merged**